### ISSUE TITLE
[BUG] .bashrc documentation incorrect.

### ISSUE THREAD

**Issue by jfujitani** [mylinuxforwork/dotfiles#327]
**Describe the bug**
The bash customization documents say to create a file that is not sourced by any of the scripts as it state.  

**To Reproduce**
1. Go to https://github.com/mylinuxforwork/dotfiles/wiki/Customize-the-.bashrc 
2. See the documentation still says to create a bashrc file that doesn't appear the included `.bashrc` will run.  I appears the latest `.bashrc` files will instead use the `~/.config/bashrc/` and `~/.config/bashrc/custom` folders to look for bash configuration.

**Expected behavior**
Not sure. I'm not certain if what the docs say or what the latest `.bashrc` files do, are the intended way of accomplishing this. 

**Screenshots**

**Distribution (please complete the following information):**
 - EndeavourOS

**Additional context**
Note I'd like to use stow to manage my `.bashrc` (or soon to be `.zshrc`) files but, I don't know if it works with the `~/.config/bashrc/` way of doing things or at least I could not get it to work.  I think the way the linked documentation says to do it might be compatible with that.

**Comment by rlovell3** [comment]
If I may add, (I have at least 6 machines running arch + various versions of ml4w) on one of my machines, the .bashrc file itself sources .bashrc_custom if it exists.  The .bashrc file even notes in a comment up top to create a .bashrc_custom file if any customizations are needed.  But on another version of ml4w, the .bashrc file comment up top says to add files to ~/.config/bashrc, but a symbolic link already exists at that location with that name, so it is impossible to create any custom config file there.  It further suggests one could create the ~/.config/zshrc/custom directory, and place customizations there, but ~/.config/zshrc is already there as a symlink to ../dotfiles/.config/zshrc, thus making that a bad choice as well.  Please, please go back to having the .bashrc file look for and source any user-created ~/.bashrc_custom.  That is the easiest way and cleanest (in my humble opinion).

In short, there is no easy way to really go about customizations in these recent versions of ml4w.

**Comment by mylinuxforwork** [comment]
> If I may add, (I have at least 6 machines running arch + various versions of ml4w) on one of my machines, the .bashrc file itself sources .bashrc_custom if it exists. The .bashrc file even notes in a comment up top to create a .bashrc_custom file if any customizations are needed. But on another version of ml4w, the .bashrc file comment up top says to add files to ~/.config/bashrc, but a symbolic link already exists at that location with that name, so it is impossible to create any custom config file there. It further suggests one could create the ~/.config/zshrc/custom directory, and place customizations there, but ~/.config/zshrc is already there as a symlink to ../dotfiles/.config/zshrc, thus making that a bad choice as well. Please, please go back to having the .bashrc file look for and source any user-created ~/.bashrc_custom. That is the easiest way and cleanest (in my humble opinion).
> 
> In short, there is no easy way to really go about customizations in these recent versions of ml4w.

Thanks for your feedback. I will look into it.

**Comment by mylinuxforwork** [comment]
@rlovell3 Just pushed an update to bring back the .zshrc_custom and .bashrc_custom support.
